### ISSUE TITLE
Swift 3 support through cocoapods

### ISSUE THREAD

**Issue by MarcelBloemendaal** [instacart/TrueTime.swift#14]
Hello, I would like to use TrueTime using Cocoapods since it looks like a great library! However, it's swift-3 branch is not accessible through cocoapods. Will this be fixed in the near future?

(This issue was edited: it previously was about the 'Result' dependency not being swift 3 compatible yet, but I discovered that was not the issue. My apologies...)

**Comment by msanders** [comment]
Thanks for the heads-up. Swift 3 support has now been merged and uploaded to an official release available on CocoaPods (4.0). Let me know what you think.

**Comment by MarcelBloemendaal** [comment]
Great, thank you! I had already noticed the cocoapods support on the swift
3 branch earlier, but I've now moved to version 4.0. It works. I'm still
working on implementing some things, but so far it works well. There are
alternatives to your library, but as far as I've seen TrueTime is the only
one that also has error handling when network time can not be retrieved.
Using TrueTime seems to be very straightforward as well. So I'm very happy
with it. Thanks again!

With regards,
Marcel Bloemendaal (Netherlands)


On 22 November 2016 at 04:20:30, Michael Sanders (notifications@github.com)
wrote:

Thanks for the heads-up. Swift 3 support has now been merged and uploaded
to an official release available on CocoaPods (4.0). Let me know what you
think.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/instacart/TrueTime.swift/issues/14#issuecomment-262137104>,
or mute the thread
